integration-tests/chopsticks/src/helper.ts

```diff
@@ -0,0 +1,8 @@
+import { Keyring } from '@polkadot/keyring'
+
+const keyring = new Keyring({ type: 'ed25519', ss58Format: 2 })
+keyring.setSS58Format(38)
```

Comment:
Why not using it directly as the parameter in the previous line?

Reply:
True: [58b6caa](https://github.com/KILTprotocol/kilt-node/pull/614/commits/58b6caa6a3cc3253a8573b878263b23f3c8c7c93)